### DISCUSSION TITLE
Youtube allow Muxed Streams of low quality 360

### DISCUSSION THREAD

**Discussion by Thorastur** [Tyrrrz/YoutubeExplode#823] [Q&A] [answered]
Youtube allow Muxed Streams of low quality 360 but i don´t know how to obtain or if you change something.
I am trying with:
 streamInfo = streamManifest.GetMuxedStreams().FirstOrDefault();
But not work, and i am sure there is 360 for this video.
You can see that muxed 360 p still working on this page https://www.freemake.com/es/free_video_downloader_good/

**Comment by Tyrrrz** [comment, marked as answer]
Unfortunately, muxed streams are not available with the client we're currently using

**Comment by klaxxon123** [comment]
you can mux it localy in realtime with vlc media player. Just open audio stream and as advanced you can pick secondary stream in this case video. Looks much better than youtube muxed streams : )

**Comment by klaxxon123** [comment]
you can also do the same with windows media player in UWP/WPF/Forms. Look here example https://stackoverflow.com/questions/51276649/how-to-play-video-and-audio-from-separate-urls-in-uwp-app
